backend/src/common/enums/express/router-params.enum.ts

```diff
@@ -0,0 +1,10 @@
+enum RouterParams {
```

Comment:
```suggestion
enum RouterParam {
```
enum should be always singular